packages/apify/src/actor.ts

```diff
@@ -834,12 +834,12 @@ export class Actor<Data extends Dictionary = Dictionary> {
             ...this.config
                 .getEventManager()
                 .listeners(EventType.PERSIST_STATE)
-                .map(async (x) => x()),
+                .map(async (x) => (x as any)({})),
```

Comment:
This is horrible. But I guess it's the right approach to fix an actual issue first and worry about making the event manager reasonably typesafe later.